packages/roosterjs-react-command-bar/lib/components/LinkDialog.tsx

```diff
@@ -90,18 +91,24 @@ class LinkDialog extends React.PureComponent<LinkDialogProps, LinkDialogState> {
     };
 
     private dismissDialog = (ev?: React.MouseEvent<HTMLButtonElement>): void => {
-        const { editor, calloutOnDismiss = NullFunction } = this.props;
-        calloutOnDismiss(ev as any);
+        const { editor, onDismiss = NullFunction } = this.props;
+        onDismiss(ev as any);
         editor && !editor.isDisposed() && editor.restoreSavedRange();
     };
 }
 
 export function createLinkDialog(doc: Document, props: LinkDialogProps): () => void {
+    const { editor, onDismiss } = props;
+
     let container = doc.createElement("div");
     doc.body.appendChild(container);
     const dispose = (): void => {
         if (container) {
             ReactDOM.unmountComponentAtNode(container);
+            // hack to clear placeholder and also to get cursor visible again in Firefox
+            container.setAttribute("tabindex", "0");
+            container.focus();
+            editor && !editor.isDisposed() && editor.focus();
```

Comment:
Are these both .focus required only for FF or all browsers?

Reply:
no, it clears the placeholder--and gets cursor visible again in FF